Sources/Sentry/Public/SentryThread.h

```diff
@@ -34,6 +34,11 @@ SENTRY_NO_INIT
  */
 @property (nonatomic, copy) NSNumber *_Nullable current;
 
+/**
+ * The thread from which this SentryThread extracted its information
+ */
+@property (nonatomic) uintptr_t thread;
```

Comment:
`h`: This is public API. We can't add this here.

Reply:
> I think we could specify in getCurrentThreadsWithStackTrace that the first thread is always the main thread.

Not an optimal solution but will work.

Reply:
Yes, indeed.